src/renderer/main.js

```diff
@@ -78,9 +85,25 @@ network.init();
 settings.sync();
 wallets.sync();
 
+
+// i18n
+const messages = {
```

Comment:
Lets put these in a constant, since this is the second place this exact list is being used (LocaleChanger.vue L14).

Reply:
These are two different lists actually, this one here is the actual messages, in LocaleChanger it just is a list of language codes to show in the dropdown.

Reply:
Either way, lets go ahead and throw them in `renderer/constants.js`. They can be accessed in a component via `this.$constants`.

Reply:
ahh ok